src/continuum_opacity/opacity_He.jl

```diff
@@ -0,0 +1,106 @@
+"""
+According to Gray (2005), the bound-free contributions from He⁻ are usually assumed to be 
+negligible because it only has one bound level with an ionization energy 19 eV. Supposedly the 
+population of that level is too small to be worth considering.
+
+We are currently missing free-free and bound free contributions from He I.
+"""
+
+using ..ContinuumOpacity: hydrogenic_bf_opacity, hydrogenic_ff_opacity, ionization_energies
+const _He_II_ion_energy = ionization_energies["He"][2] # not sure if this is a good idea
+
+He_II_bf(nHe_II_div_partition, ν, ρ, T, ion_energy = _He_ion_energy) =
+    hydrogenic_bf_opacity(2, 9, nHe_II_div_partition, ν, ρ, T, ion_energy)
+He_II_ff(nHe_II, ne, ν, T) = hydrogenic_ff_opacity(2, nHe_II, ne, ν, T)
+
+
+# Compute the number density of atoms in different He I states
+# taken from section 5.5 of Kurucz (1970)
+function ndens_state_He_I(n::Integer, nsdens_div_partition::Flt, T::Flt) where {Flt<:AbstractFloat}
+    if n == 1
+        g_n, energy_level = (1.0, 0.0)
+    elseif n == 2
+        g_n, energy_level = (3.0, 19.819)
+    elseif n == 3
+        g_n, energy_level = (1.0, 20.615)
+    elseif n == 4
+        g_n, energy_level = (9.0, 20.964)
+    else
```

Comment:
I think you could factor out the assignment: `g_n, energy_level = if ...`

Reply:
good point